nbodykit/tests/test_bianchi.py

```diff
@@ -0,0 +1,57 @@
+from mpi4py_test import MPITest
+from nbodykit.lab import *
+from nbodykit import setup_logging
+from numpy.testing import assert_allclose
+
+# debug logging
+setup_logging("debug")
+
+@MPITest([1, 4])
+def test_bianchi(comm):
+    
+    NDATA = 1000
+    NBAR = 1e-4
+    
+    CurrentMPIComm.set(comm)
+    cosmo = cosmology.Planck15
+    
+    data = Source.RandomParticles(NDATA, seed=42)
+    randoms = Source.RandomParticles(NDATA*10, seed=84)
+    
+    # add the random columns
+    for s in [data, randoms]:
```

Comment:
Yes, this will work I think. Multiple ranks will generate the same samples in the overlap regions, but that waste is probably unavoidable. I'll code this up